### ISSUE TITLE
colour analyzer doppler geting big - on bigsur MAC

### ISSUE THREAD

**Issue by rashmitej** [ThePacielloGroup/CCAe#212]
**Describe the bug**

**Desktop (please complete the following information):**
 - OS: [e.g. Windows 10 32 bits]
 - Version [e.g. 1.0.0]

**To Reproduce**

**Expected behavior**

**Screenshots**

**Additional context**

**Comment by rashmitej** [comment]
![image](https://user-images.githubusercontent.com/64445362/121089814-aff4dc00-c7b5-11eb-83b7-321f62989640.png)

**Comment by ferllings** [comment]
Hello @rashmitej ,
I'm not sure to understand what your problem is: By default the color picker is zoomed to facilitate the selection of a single pixel.

**Comment by tejeswir** [comment]
yes @ferllings . I installed Colour analyzer and by default color picker is zoomed. (I checked in some of the  previously created issues similar color picker zoomed issue in windows below comments https://github.com/ThePacielloGroup/CCAe/issues/201)

**Comment by ferllings** [comment]
@tejeswir no this is not the same. The user had a problem with the picker displaying with an offset.

I this case I'm still unsure about the problem, as it seems to work as expected.

> "doppler size should be small"
Why? the picker zooming factor is there to ensure you're selecting the right pixel color.

**Comment by rashmitej** [comment]
Sorry for any confusion @ferllings .. The Problem is I see the doppler zoomed, When I installed on my Mac.

**Comment by lechubb** [comment]
I am experiencing a similar issue. I am unable to use CCA because the eyedropper will not let me choose the color that is "under" the dropper (or even the general area I am aiming for). The magnifier will not accurately display what is "under it" unless it is the very top left corner—which is obviously not what I need it for. I'm on a MacBook Pro and have attempted to use CCA in InDesign and Acrobat. 
Attaching screenshots.
![image](https://user-images.githubusercontent.com/88496157/128361420-a8e0d0e9-5c03-4910-836a-16f4d7c815c7.png)
![image](https://user-images.githubusercontent.com/88496157/128361460-9f36701a-0a63-43e2-a918-2213e21cffc5.png)
![image](https://user-images.githubusercontent.com/88496157/128361504-3855aee6-ed18-4932-95fb-1d47ba74b6f4.png)
